sewer/dns_providers/auroradns.py

```diff
@@ -0,0 +1,80 @@
+# DNS Provider for AuroRa DNS from the dutch hosting provider pcextreme
+# https://www.pcextreme.nl/aurora/dns
+# Aurora uses libcloud from apache
+# https://libcloud.apache.org/
+
+from libcloud.dns.providers import get_driver
+from libcloud.dns.types import Provider, RecordType
+import tldextract
+from structlog import get_logger
+from . import common
+
+
+class AuroraDns(common.BaseDns):
+    """
+    """
+
+    def __init__(self, AURORA_API_KEY, AURORA_SECRET_KEY):
+
+        self.AURORA_API_KEY = AURORA_API_KEY
+        self.AURORA_SECRET_KEY = AURORA_SECRET_KEY
+        self.dns_provider_name = 'aurora'
+
+        self.logger = get_logger(__name__).bind(
+            dns_provider_name=self.dns_provider_name)
+
+    def create_dns_record(self, domain_name, base64_of_acme_keyauthorization):
+        self.logger.info('create_dns_record')
+
+        # delete any prior existing DNS authorizations that may exist already
+        self.delete_dns_record(
+            domain_name=domain_name,
+            base64_of_acme_keyauthorization=base64_of_acme_keyauthorization)
+
+        extractedDomain = tldextract.extract(domain_name)
```

Comment:
@wilfriedjonker ok, Thanks for the explanation. 
The only remaining issue is then; https://github.com/komuW/sewer/pull/38#issuecomment-318918496